content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -944,14 +930,11 @@ network-attached storage. See
 The following volume plugins support raw block volumes, including dynamic provisioning where
 applicable:
 
-* AWSElasticBlockStore
-* AzureDisk
 * CSI
 * FC (Fibre Channel)
 * GCEPersistentDisk
```

Comment:
Remove

Reply:
According to https://github.com/kubernetes/website/pull/42302#discussion_r1282723644, GCE PD is still in Kubernetes 1.27